index.js

```diff
@@ -16,35 +16,190 @@ const forbiddenModules = [
+    // This makes life harder if you're linting bin files that aren't transpiled
+    // or have the trailing function (ES2017 proposal) config set.
+    // This is bad as that adds friction to linting all files, which is desired.
+    'comma-dangle': [
+      'error',
+      {
+        arrays: 'always-multiline',
+        objects: 'always-multiline',
+        imports: 'always-multiline',
+        exports: 'always-multiline',
+        functions: 'never',
+      },
+    ],
+
+    // airbnb defaults to a maximum cyclomatic complexity of 11
+    // https://eslint.org/docs/rules/complexity
+    complexity: ['error'],
+
+    // Not all arrow functions need to return
+    // https://eslint.org/docs/rules/consistent-return
+    'consistent-return': 'off',
+
+    // This rule isn't ready yet
+    // Arrow functions are forced to put the return values on a new line, rather than inline
+    // with the argument and fat arrow itself, which is awkward syntax
+    // This rule, when fixing code will then cause problems with the max-length rule
+    // https://github.com/airbnb/javascript/issues/1584#issuecomment-335676788
+    'function-paren-newline': 'off',
+
+    // Prettier handles this
+    // https://eslint.org/docs/rules/implicit-arrow-linebreak
+    'implicit-arrow-linebreak': 'off',
+
+    // Prettier handles indent
+    // https://eslint.org/docs/rules/indent
+    indent: 'off',
+    // https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-indent.md
+    'react/jsx-indent': 'off',
+
+    // Functions that take many positional arguments can be difficult to work
+    // with and produce less maintainable APIs. When more than three arguments
+    // are needed, named arguments should be used.
+    'max-params': ['error', 3],
+
+    // Knowing how to use bitwise operators is rare and people who don't know how never will
+    // For the few instances where they are useful, this rule is a nuisance, so it's unnecessary
+    // https://eslint.org/docs/rules/no-bitwise
+    'no-bitwise': 'off',
+
+    // Simplifies code and reduces bugs
+    // https://eslint.org/docs/rules/no-lonely-if
+    'no-lonely-if': ['error'],
+
+    // Restrict certain modules from being used
+    'no-restricted-modules': ['error', {paths: forbiddenModules}],
+    'no-restricted-imports': ['error', {paths: forbiddenModules}],
+
+    // Reduces careless typo mistakes
+    // https://eslint.org/docs/rules/no-return-assign
+    'no-return-assign': ['error'],
+
+    // Prettier handles this
+    // https://eslint.org/docs/rules/object-curly-spacing
+    'object-curly-spacing': 'off',
+
+    // Spreading is better than complex Object.assign
+    // https://eslint.org/docs/rules/prefer-object-spread
+    'prefer-object-spread': ['error'],
+
+    // Alphabetical sorting is the most reliable way to make code consistently readable
+    // This goes for imports, props, objects, etc.
+
+    // This one is warn so it isn't disruptive during development
```

Comment:
I'm not opposed to this rule in theory, but I have concerns because:
a) it directly contradicts our previous import order rule (which was "builtin", "external", "internal")
b) there is no auto-fix for some reason

As a practical matter, I'm not sure what the path would be to getting rid of these warnings in every single existing file.

Reply:
VSCode and WebStorm both have the ability to fix import orders to match what is in eslint, even if eslint --fix itself cannot do it. For me, fixing import orders is quick business with move line up/down shortcuts.